sql/index.go

```diff
@@ -93,8 +95,9 @@ type IndexLookup struct {
 	// values; the range is null safe, the index is unique, every index
 	// column has a range expression, and every range expression is an
 	// exact equality.
-	IsPointLookup bool
-	IsEmptyRange  bool
+	IsPointLookup   bool
+	IsEmptyRange    bool
+	IsSpatialLookup bool
```

Comment:
are there any details about spatial index lookups that are not obvious any maybe helpful to document?